ReSharper.FSharp/src/FSharp.Psi/src/FSharp.psi

```diff
@@ -1670,6 +1670,9 @@ anonRecordField:
   COLON<FSHARP_COLON, Colon>
   typeUsage<TYPE, TypeUsage>;
 
+xmlDocBlock:
+  LPAREN<LPAREN, LParen>;
```

Comment:
Why?

Reply:
Generally speaking, we should make this block not in grammar at all